source/includes/_errors.md

```diff
@@ -8,6 +8,7 @@ Error Code | Meaning
 200 | OK -- Everything worked as expected.
 400 | Bad Request -- The request was unacceptable, often due to missing a required parameter.
 401 | Unauthorized -- No valid API key provided.
+402 | Not enabled -- Please contact sales before creating this type of task.
```

Comment:
Could you include the sales email address in this description? That would make it easier for customers to email us, as they don't need to do the extra step of looking up the sales email address.